adal/src/main/java/com/microsoft/aad/adal/ADALError.java

```diff
@@ -554,7 +554,7 @@
     
     private String mDescription;
 
-    private ADALError(String message) {
+    ADALError(String message) {
```

Comment:
Ok, so for enum class, no modifier means private. Then I'm good. 
I don't like the implication in the code, no one else except this class should be able initialize the enum. Java already did for us, hah, that's one android studio thinks it doesn't make sense. 

---
In reply to: [65659282](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65659282) [](ancestors = 65659282)